prince-curie/Epic-mail
# [ch-#164352192] merging the feature branch to the gh-page branch

#### What does this PR do?
- Merges the feature branch to the gh-page branch
#### How should this be manually tested?
- check gh-page branch to confirm if it has the same content with the feature branch
#### Any background context you want to provide?
- The gh-page branch is where the UI is hosted from
#### What are the relevant pivotal tracker stories?
- #164352192
